python/MooseDocs/extensions/MooseTextPatternBase.py

```diff
@@ -29,26 +29,20 @@ def __init__(self, pattern, language=None, repo=None, root=None, **kwargs):
         self._repo = repo
 
         # The default settings
-        self._settings = {'strip_header':True,
-                          'repo_link':True,
-                          'label':True,
-                          'method':True,
-                          'block':True,
-                          'overflow-y':'scroll',
-                          'max-height':'500px',
-                          'strip-extra-newlines':False}
-
-    def style(self, *keys):
-        """
-        Extract the html style string from a list of settings.
-
-        Args:
-            *keys[str]: A list of keys to compose into a style string.
-        """
-        style = []
-        for k in keys:
-            style.append('{}:{}'.format(k, self._settings[k]))
-        return ';'.join(style)
+        self._settings = {'strip_header'        : True,
+                          'repo_link'           : True,
+                          'label'               : True,
+                          'method'              : True,
+                          'block'               : True,
+                          'strip-extra-newlines': False}
+
+        # We need a way to limit CSS to elements that we know causes issues.
+        # example: to ignore overflow-y to only div elements
+        # self.invalid_css = { 'div' : ['overflow-y'] }
+        #
```

Comment:
You can remove this comment, this is what should be in the base MooseCommonExtension. The comment below this is good.